packages/backend/src/database/migrations/20210910130905_add_credentials_table.ts

```diff
@@ -0,0 +1,45 @@
+import { Knex } from 'knex';
+import { WarehouseCredentialTableName } from '../entities/warehouseCredentials';
```

Comment:
Not sure how I feel about imports in migration files. I usually think about them as pure functions, having no dependencies ( beside knex ).

Reply:
👍